release_files/Dockerfile.ubuntu

```diff
@@ -153,6 +153,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends libxml2 && rm -
 COPY --from=base_build /usr/local/lib/ /ovms/lib/
 COPY --from=base_build /ovms /ovms
 
+# Remove capi library
+RUN if [ -f /ovms/lib/libovms_shared.so ] ; then true ; else exit 0 ; fi ;\
```

Comment:
move it to the first stage of the build and simplify the command. else is not needed here

Reply:
Cant move earlier , it just has been copied from /ovms.

Reply:
you should do it in between line 62 and 63